src/components/board.js

```diff
@@ -21,6 +21,37 @@ export default class MainBoard extends Core {
     return this.template;
   }
 
+  handleDragOver(event) {
+    event.preventDefault();
+    this.setAttribute('active', '');
+    let found;
+
+    if (!this.draggingElement) {
+      found = event.composedPath().find((node) => node.className === 'card' && node.nodeName !== 'SLOT');
+
+      if (found) {
```

Comment:
```suggestion
    if(!found) return; 
```
p4:  이렇게 바꾸면 아래 로직을 조건문안에 안넣어도 될 것 같네요~